UnityProject/Assets/Scripts/Core/Input System/InteractionV2/TileInteraction/PipeDeconstruction.cs

```diff
@@ -50,13 +59,12 @@ public override void ServerPerformInteraction(TileApply interaction)
 			{
 				if (metaDataNode.PipeData[i].RelatedTile != pipeTile) continue; //TODO Stuff like layers and stuff can be included
 
-				ToolUtils.ServerUseToolWithActionMessages(interaction, 1f,
-						"You start to deconstruct the pipe..",
-						$"{interaction.Performer.ExpensiveName()} starts to deconstruct the pipe...",
-						"You deconstruct the pipe",
-						$"{interaction.Performer.ExpensiveName()} deconstructs the pipe.",
-						() => { metaDataNode.PipeData[i].pipeData.Remove(); });
-
+				ToolUtils.ServerUseToolWithActionMessages(interaction, seconds,
+					performerStartActionMessage,
+					Chat.ReplacePerformer(othersStartActionMessage, interaction.Performer),
```

Comment:
Does this handle null strings nicely or will it throw?

Reply:
This just calls AddActionMsgToChat which does call IsNullOrWhitespace on the interaction messages